### DISCUSSION TITLE
Filter results based on HTTP body content

### DISCUSSION THREAD

**Discussion by thefirst100yearsaredifficult** [cea-sec/ivre#1533] [Q&A] [answered]
Hello,
first of all very cool project. I want to ask if there are filters such as “http.html” from shodan. If yes what would be the nmap command for scanning and later importing it?

Kind regards
Thanks for your time

**Comment by p-l-** [comment, marked as answer]
For now, only Masscan, Zgrab2 & Dismap are supported for HTTP content (I don't think an Nmap script exists for that).

The data is contained in a "script" called `http-content`. So you can use `script:http-content:/test/i` for example. Please note that this will be slow if you have a lot of results, but you can help IVRE (& MongoDB) by adding an **additional** filter `search:text`.

`search:text` will use the full text index in all the text fields of the results, and `script:http-content:/test/i` will limit the results to those that have `"text"` in the HTTP content.

Hope this helps!
